packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -647,9 +647,10 @@ export async function resetAndUpgradeHelmChart(celoEnv: string) {
   await scaleResource(celoEnv, 'Deployment', bootnodeName, 0)
 
   await deletePersistentVolumeClaims(celoEnv)
-  await sleep(5000)
+  await sleep(10000)
```

Comment:
Can you de-duplicate this 10,000 milis-seconds into one constant?

Reply:
I feel like there's no reason they need to be the same, as they're waiting on different things -- hence a single constant doesn't make sense